packages/components/src/tooltip/index.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { useRef, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { createPopper } from '@popperjs/core';
+import { getPopperOptions } from './utils';
+import styles from './index.module.scss';
+
+const Tooltip = (props) => {
+  const {
+    type,
+    bubble,
+    children,
+    tooltipClassName,
+    arrowClassName,
+    triggerProps,
+    className,
+    fallbackPosition,
+  } = props;
+
+  const content = useRef(null);
+  const tooltip = useRef(null);
+  const instance = useRef(null);
+  const arrow = useRef(null);
+
+  const mountPopper = () => {
+    tooltip.current.setAttribute('data-show', '');
+    instance.current = createPopper(
+      content.current,
+      tooltip.current,
+      getPopperOptions(arrow.current, type, fallbackPosition),
+    );
+  };
+
+  const destroyPopper = () => {
+    tooltip.current.removeAttribute('data-show');
+    if (instance.current) instance.current.destroy();
+  };
+
+  useEffect(() => {
+    mountPopper();
+    return () => {
+      destroyPopper();
+    };
+  }, [children]);
```

Comment:
brave move - did you test it with a value that changes every second? is it even needed?